frontend/cypress/integration/common/sidecar_injection.ts

```diff
@@ -0,0 +1,274 @@
+import { Given, Then, When } from "cypress-cucumber-preprocessor/steps";
+
+Given('a namespace without override configuration for automatic sidecar injection', function () {
+    this.targetNamespace = "default";
+
+    // Make sure that the target namespace does not have override configuration
+    cy.request('PATCH', '/api/namespaces/' + this.targetNamespace, {
```

Comment:
I'm not sure if I like this.
You are using the implementation detail when the test should focus on the UI.